src/WebJobs.Script/Binding/Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -40,7 +40,7 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             // Can bind to user types, HttpRequestMessage, object (for dynamic binding support) and all the Read
             // Types supported by StreamValueBinder
             IEnumerable<Type> supportedTypes = StreamValueBinder.GetSupportedTypes(FileAccess.Read)
-                .Union(new Type[] { typeof(HttpRequestMessage), typeof(object) });
+                .Union(new Type[] { typeof(HttpResponseMessage), typeof(HttpRequestMessage), typeof(object) });
```

Comment:
This addition would apply to regular http bindings where it doesn't make sense to bind to a response. We don't want to allow that for regular http.